babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -342,4 +342,9 @@ private void checkParseInfixCast(String sqlType) {
     sql(sql).ok(expected);
   }
 
+  @Test public void testCastFormat() {
```

Comment:
This is a great test case. But I suggest we add at least one other type as well. For example, something like:

CAST('2020-06-02' AS DATE FORMAT 'yyyy-mm-dd')